src/app/core/entity/entity-type-selector/entity-type-selector.component.html

```diff
@@ -0,0 +1,9 @@
+<mat-form-field>
```

Comment:
Do we need this mat-form-field to wrap around it here?

Reply:
I think yes, we need the`mat-form-field` to apply both styling and behaviour to the form controls.

Reply:
I understand. But I think it may be better here to put the mat-form-field wherever we use app-entity-type-select, instead of inside the component itself. That's the model we chose for app-basic-autocomplete also (and I hope we had some reasons for that :sweat_smile: ). This would also ensure the EditEntityTypeDropdown will define its own mat-form-field including label, which you have currently lost now